examples/jenkins/README.md

```diff
@@ -68,8 +68,10 @@ Steps
 6. Create the sample application configuration
 
         $ oc new-app application-template.json
+
+    Note the generated password (JENKINS_PASSWORD) reported in the new-app output, you will need it to login to Jenkins.
```

Comment:
Should you include directions on how to get it if creating from the web console?